apps/forklift/lib/forklift/data_reader_helper.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Forklift.DataReaderHelper do
```

Comment:
Does this need to be "DataReaderHelper" or can it just be "DataReader" = "the forklift implementation of the pipeline data reader"?

Reply:
We thought about that, but it's technically not an implementation of the behavior so we wanted to make that clear(er)

Reply:
That’s fair. I just don’t like the “Helper” part. For some reason it feels like a bolt-on